### DISCUSSION TITLE
Not working with Archer T2U Plus

### DISCUSSION THREAD

**Discussion by luca-mkbr** [chris1111/Wireless-USB-Big-Sur-Adapter#38] [Q&A]
Hi, I tried installing your adapter on my hackintosh running Big Sur but the tool says “No USB-WiFi Plug in”. I have tried other USB ports and that did not help. When I extract the zip file though, I get this error that says: “Unable to expand ‘Wireless USB OC Big Sur Adapter-V6” “(Error 32 - Broken Pipe.)”. I had no idea what this error meant but I googled it and saw mentions of timeouts. Do need to be connected to the internet to install the adapter?

**Comment by chris1111** [comment]
> Hi, I tried installing your adapter on my hackintosh running Big Sur but the tool says “No USB-WiFi Plug in”. I have tried other USB ports and that did not help. When I extract the zip file though, I get this error that says: “Unable to expand ‘Wireless USB OC Big Sur Adapter-V6” “(Error 32 - Broken Pipe.)”. I had no idea what this error meant but I googled it and saw mentions of timeouts. Do need to be connected to the internet to install the adapter?


This error is come from Internet browser use safari then download in .pkg file
ARCHER T2U Plus is already test and its a known working adapter
so check your setup Hackintosh
